src/client/datascience/notebook/integration.ts

```diff
@@ -5,9 +5,12 @@ import { inject, injectable } from 'inversify';
 import { languages } from 'vscode';
 import { IExtensionSingleActivationService } from '../../activation/types';
 import { NOTEBOOK_SELECTOR } from '../../common/constants';
-import { IDisposableRegistry } from '../../common/types';
+import { IConfigurationService, IDisposableRegistry } from '../../common/types';
 import { JupyterCompletionProvider } from './intellisense/jupyterCompletionProvider';
 
+// Default set of trigger characters for jupyter
+const DefaultTriggerCharacters = ['.', ',', '%'];
```

Comment:
Different from the default value in settings

Reply:
Oops